build/x86_64_debian/Dockerfile

```diff
@@ -1,33 +1,10 @@
-FROM debian:7
-
-RUN \
- # Point to the archive...
-    sed -i \
-      -e 's/\(deb\|httpredir\)\.debian\.org/archive.debian.org/g' \
-      -e '/-updates main/d' \
-      -e '/http\.debian\.net/d' \
-      /etc/apt/sources.list \
- # ...and disable the expiration check...
- && echo 'Acquire::Check-Valid-Until "0";' \
-      >/etc/apt/apt.conf.d/10no-check-valid-until \
- # ...and remove pointers to the defunct security repos.
- && sed -i \
-      -e '/security\.debian\.org/d' \
-      /etc/apt/sources.list
+FROM debian:9
```

Comment:
Have we verified that a package built in a Debian 9 image works in Ubuntu Xenial end others?

Reply:
See the internal ticket for related testing.